app/views/application/post.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card my-4">
+    <div class="card-header">
+        <a href="/users/<%=user.id%>/posts/<%=post.id%>"><h4># <%= post.Title%></h4></a>
+    </div>
+    <div class="card-body"><%=post.text%></div>
+    <div class="media-info">
+        <ul>
+            <li><a href="#">Comments: <%=post.comments_count%></a></li>
```

Comment:
- [ ] Instead of `comments_count` it should be `comments_counter` and because of this, an error is being thrown and the method cannot be recognized.
![image](https://github.com/maths29/iwuBlog/assets/45460192/9a6a504a-a07a-4796-89b6-3cc1e2654b13)